app/konnect/service-catalog/index.md

```diff
@@ -0,0 +1,126 @@
+  classDef billingInventory fill:#f0f0f0,stroke:#333,stroke-width:2px;
+
+  %% Define the resources
+  GitHub[<img src="https://img.icons8.com/ios/50/000000/github.png" width="24" height="24"/> GitHub]:::resource
+  PagerDuty[<img src="https://play-lh.googleusercontent.com/E-zhAf4KJ6JDDXmQfQxBprn2sATGYUMkOEqLQX5HAQQtiwDZJg4c8sQd7deb6nCZCwU=w480-h960-rw" width="24" height="24"/> PagerDuty]:::resource
+  KongGateway[<img src="https://raw.githubusercontent.com/Kong/docs.konghq.com/main/app/assets/images/icons/kong-gradient.svg" width="15" height="15"/> Kong Gateway]:::resource
+
+  %% Define the Konnect Service Catalog and its subgraphs
+  subgraph KonnectServiceCatalog[Konnect Service Catalog]
+    direction TB
+
+    %% Define the Billing Service subgraph inside Konnect Service Catalog
+    subgraph BillingService[Billing Service]
+      direction LR
+      KongGatewayService[Kong Gateway service 1]
+      PagerDutyServiceMapping[PagerDuty service 1]
+      GitHubRepoMapping[GitHub repository 1]
+    end
+
+    %% Define the Inventory Service Catalog Service subgraph inside Konnect Service Catalog
+    subgraph InventoryService[Inventory Service]
+      direction LR
+      SomeOtherService[Kong Gateway service 2]
+      AnotherService[PagerDuty service 2]
+      DifferentRepo[GitHub repository 2]
+    end
+
+  end
+
+  %% Connect the resources to the Konnect Service Catalog
+  GitHub -- "GitHub repositories" --> KonnectServiceCatalog
+  PagerDuty -- "PagerDuty services" --> KonnectServiceCatalog
+  KongGateway -- "Kong Gateway services" --> KonnectServiceCatalog
+
+  %% Style the subgraphs
+  class KonnectServiceCatalog konnect
+  class BillingService serviceCatalog
+  class InventoryService serviceCatalog
+{% endmermaid %}
+<!-- vale on-->
+
+> Figure 1: This diagram shows how you can use both external integrations, like GitHub and PagerDuty, as well as built-in integrations like {{site.base_gateway}} to pull resources into Service Catalog. You can then map those resources (like GitHub repositories, PagerDuty services, and {{site.base_gateway}} services) to Service Catalog Services.
+
+## Service Catalog use cases
+
+| You want to... | Then use... |
+| -------------- | ----------- |
+| Keep track of your organization's service ownership by mapping teams to services. | [{{site.konnect_short_name}} Organization Teams](https://cloud.konghq.comorganization/teams) |
+| Gain visibility into all your services, including unrecognized or undiscovered APIs in your organization.  | [Service Catalog integrations](https://cloud.konghq.com/servicehub/integrations) |
+| Consolidate key service health metrics into a single dashboard, allowing you to interact with other tools from one place. | [Service Catalog dashboard](https://cloud.konghq.com/servicehub) |
+| Govern how services are created and maintained across your company to adhere to security, compliance, and engineering best practices. | ? |
```

Comment:
@miko-bautista I wasn't sure which Service Catalog feature allowed users to create a governance policy.

Reply:
That would be Scorecards! Here is the PRD: https://docs.google.com/document/d/1vyUihjEJ_4qHIVhIn2TzSdg0yi7FgsvPAsXk6-nclOY/edit#heading=h.xkhrhr69reo5

During our call tomorrow, I can walk you through some of Julieta's early Figma mockups for this feature and answer any questions you might have.